src/sonic-yang-models/yang-models/sonic-fabric-monitor.yang

```diff
@@ -43,6 +43,19 @@ module sonic-fabric-monitor{
                     description "Consecutive polls with lesser error rate for inclusion.";
                 }
 
+                leaf monCapacityThreshWarn {
+                    type uint32;
```

Comment:
the max capacity is 100%, do we need uint32 for this?

Reply:
> the max capacity is 100%, do we need uint32 for this?

revised.